cmd/mmforc/Dockerfile

```diff
@@ -1,28 +1,9 @@
-# Golang application builder steps
-FROM golang:1.11.5 as builder 
-
-# Necessary to get a specific version of the golang k8s client
-RUN go get github.com/tools/godep
-RUN go get k8s.io/client-go/...
-WORKDIR /go/src/k8s.io/client-go
-RUN git checkout v7.0.0
-RUN godep restore ./...
-RUN rm -rf vendor/
-RUN rm -rf /go/src/github.com/golang/protobuf/
-
-# Get only the necessary files from the repo.
-# https://stackoverflow.com/a/39317180/3113674
-# TODO: Set these to the final branch name after RC approval.
-WORKDIR /go/src/github.com/GoogleCloudPlatform/open-match
-RUN svn export https://github.com/GoogleCloudPlatform/open-match/branches/040wip/internal
-RUN svn export https://github.com/GoogleCloudPlatform/open-match/branches/040wip/config
+FROM open-match-base-build as builder
 
 WORKDIR /go/src/github.com/GoogleCloudPlatform/open-match/cmd/mmforc/
-COPY . .
-RUN go get -d -v 
 RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo .
 
-FROM gcr.io/distroless/static
+FROM gcr.io/distroless/static  
```

Comment:
super nit :) Extra space at line end? (Likewise for the rest of the places this has been added in this file)